src/schema/nodes/equation.ts

```diff
@@ -17,64 +17,63 @@
 import { ObjectTypes } from '@manuscripts/json-schema'
 import { NodeSpec } from 'prosemirror-model'
 
+import { convertTeXToMathML } from '../../mathjax/tex-to-mathml'
 import { ManuscriptNode } from '../types'
 
 interface Attrs {
```

Comment:
Let's add a "format" attribute to both this and inline_equation (and json-schema). The allowed values should be "tex" and "mathml".